examples/chapter1/src/example3-jlbarros.cpp

```diff
@@ -0,0 +1,71 @@
+#include <algorithm>
+#include <iostream>
+#include <thread>
+
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
+
+const int poolSize{16};
+const int vectorSize{3};
+
+int main(int argc, char *argv[])
```

Comment:
Same as before, make it relevant and didactic by combining different configurations of the publishers, in this case with lambda functions rather than a class.